---
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}



## Academic publications

**Claire Duvallet**, Kara Larson, Scott Snapper, Sonia Iosim, Ann Lee, Katherine Freer, Kara May, Eric Alm, and Rachel Rosen. "Aerodigestive sampling reveals altered microbial exchange between lung, oropharyngeal, and gastric microbiomes in children with impaired swallow function." _bioRxiv_ (2018) doi: [10.1101/476580](https://www.biorxiv.org/content/early/2018/11/23/476580).
Associated github: [cduvallet/aspiration-analysis-public](https://github.com/cduvallet/aspiration-analysis-public)

Keegan Korthauer\*, Patrick K Kimes\*, **Claire Duvallet**^, Alejandro Reyes^, Ayshwarya Subramanian^, Mingxiang Teng, Chinmay Shukla, Eric J Alm, and Stephanie C Hicks. "A practical guide to methods controlling false discoveries in computational biology." _bioRxiv_ (2018) doi: [10.1101/458786](https://www.biorxiv.org/content/early/2018/10/31/458786).
Associated github: [pkimes/benchmark-fdr](https://github.com/pkimes/benchmark-fdr)

Thomas Gurry, HST Microbiome Consortium, Sean M Gibbons, Sean M Kearney, Ashwin Ananthakrishnan, Xiaofang Jiang, **Claire Duvallet**, Zain Kassam, and Eric J Alm. "Predictability and persistence of prebiotic dietary supplementation in a healthy human cohort." _Scientific Reports_ (2018) 8(1): 12699. doi: [10.1038/s41598-018-30783-1](https://doi.org/10.1038/s41598-018-30783-1)

Sean M. Gibbons, **Claire Duvallet**, and Eric J. Alm. "Correcting for batch effects in case-control microbiome studies." _PLoS Computational Biology_ (2018) 14(4): e1006102. doi: [10.1371/journal.pone.0176335](https://doi.org/10.1371/journal.pcbi.1006102)

**Claire Duvallet**. "Meta-analysis generates and prioritizes hypotheses for translational microbiome research." _Microbial Biotechnology_ (2018). doi: [10.1111/1751-7915.13047](https://doi.org/10.1111/1751-7915.13047)

**Claire Duvallet**, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm. "Meta-analysis of gut microbiome studies identifies disease-specific and shared responses." _Nature Communications_ (2017). doi: [10.1038/s41467-017-01973-8](https://doi.org/10.1038/s41467-017-01973-8). Associated github: [cduvallet/microbiomeHD](https://github.com/cduvallet/microbiomeHD) and data: [zenodo record 840333](https://doi.org/10.5281/zenodo.797943).

Scott W. Olesen, **Claire Duvallet**, and Eric J. Alm. "dbOTU3: A new implementation of distribution-based OTU calling." _PloS ONE_ 12, no. 5 (2017): e0176335. doi: [10.1371/journal.pone.0176335](https://doi.org/10.1371/journal.pone.0176335)

## Non-peer reviewed academic content

_[Blog post]_ **Claire Duvallet**. "[Beyond dysbiosis: disease-specific and shared microbiome responses to disease.](https://naturemicrobiologycommunity.nature.com/users/70264-claire-duvallet/posts/22494-beyond-dysbiosis-disease-specific-and-shared-microbiome-responses-to-disease)" _Nature Microbiology Community Forum._ December 2017.

_[Dataset]_ **Claire Duvallet**, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm. (2017). MicrobiomeHD: the human gut microbiome in health and disease. _Zenodo_. doi: [10.5281/zenodo.797943](https://doi.org/10.5281/zenodo.797943)

## Software

**Percentile normalization**
Correcting batch effects in case-control microbiome studies. (_Gibbons et al., 2018_)

* Python implementation: [seangibbons/percentile_normalization](https://github.com/seangibbons/percentile_normalization) _(contributor)_
* [QIIME 2](https://qiime2.org/) plugin: [cduvallet/q2-perc-norm](https://github.com/cduvallet/q2-perc-norm) _(developer)_

**Distribution-based OTU calling**
New implementation of Preheim _et al._'s distribution-based OTU clustering algorithm. (_Preheim et al., 2013_; _Olesen et al., 2017_).

* Python implementation: [swo/dbotu3](https://github.com/swo/dbotu3) _(contributor)_
* [QIIME 2](https://qiime2.org/) plugin: [cduvallet/q2-dbotu](https://github.com/cduvallet/q2-dbotu) _(developer)_

**Amplicon sequencing pipeline**
End-to-end pipeline to process 16S data.

* Python: [thomasgurry/amplicon_sequencing_pipeline](https://github.com/thomasgurry/amplicon_sequencing_pipeline) _(co-developer)_

## Other

_[Resource]_ **Claire Duvallet** and Monika Avello. "[Graduate student support resources flowchart.](/files/2018-04-01-Grad-Support-Flowchart-MIT-Digital.pdf )" I and another student on [iREFS](http://refs.mit.edu/irefs/) created this flowchart to help graduate students find support at MIT.

_[Blog posts]_ **Claire Duvallet**. During my Luce year (2014-2015), I wrote some posts for the Engineering World Health [blog](http://www.ewh.org/news/blog):

* [Turning Knowledge into Practice: Cambodia BMET Training](http://www.ewh.org/news/blog/463-turning-knowledge-into-practice-cambodia-bmet-training)
* [Pass It On: Teaching in Cambodia](http://www.ewh.org/news/blog/472-pass-it-on-teaching-in-cambodia)
* [A Late Night In Phnom Penh](http://www.ewh.org/news/blog/477-a-late-night-in-phnom-penh)
* [Final Exams and Weddings in Cambodia](http://www.ewh.org/news/blog/489-final-exams-and-weddings-in-cambodia)
* [BMET Bonding in Phnom Penh](http://www.ewh.org/news/blog/508-bmet-bonding-in-phnom-penh)
* [BMET Cambodia Welcomes Dhritiman Das!](http://www.ewh.org/news/blog/522-bmet-cambodia-welcome-dhritiman-das)
